apps/akka-kafka-streams-model-server/source/core/akkastreamssvc/src/main/resources/grafana-source.json.template

```diff
@@ -3,11 +3,11 @@
   "type": "influxdb",
   "typeLogoUrl": "public/app/plugins/datasource/influxdb/img/influxdb_logo.svg",
   "access": "proxy",
-  "url": "http://influxdb.marathon.l4lb.thisdcos.directory:8086",
+  "url": "http://INFLUXDB_HOST:8086",
   "password": "root",
   "user": "root",
   "database": "serving",
   "basicAuth": true,
   "isDefault": false,
   "jsonData": {}
-}
```

Comment:
This is not template this file is processed by the code to produce the right thing. No change is necessary